locale/en/messages.json

```diff
@@ -786,7 +795,7 @@
   "There are no plans currently available.": "",
   "There are tasks in this queue that DC/OS has failed to deploy for {duration}.": "",
   "There is an error with your configuration": "",
-  "This action <0>CANNOT</0> be undone. This will permanently delete the <1>{serviceName}</1> {0}.": "",
+  "This action <0>CANNOT</0> be undone. This will permanently delete the <1>{serviceName}</1> {0} .": "",
```

Comment:
Are these prettier artifacts? 😭

Reply:
Yes, these are Prettier artifacts. I'll try and figure something out so we're not breaking translations